randovania/layout/layout_description.py

```diff
@@ -108,14 +109,18 @@ def create_new(
             generator_parameters=generator_parameters,
             all_patches=all_patches,
             item_order=item_order,
+            user_modified=False,
         )
 
     @classmethod
     def from_json_dict(cls, json_dict: dict) -> typing.Self:
+        expected_checksum = json_dict.pop("checksum", None)
```

Comment:
I thought about what's the benefit of using `None` as default here because it leads to all current rdvgames being seen as `user_modified` (as the commited patcher files shows).
But I guess both is fine.

Reply:
I'd rather consider old rdvgames as modified than have a trivial way of marking a file as not modified.